chrome/content/zotero/itemTreeColumns.jsx

```diff
@@ -267,6 +267,14 @@ const COLUMNS = [
 		flex: 1,
 		zoteroPersist: new Set(["width", "hidden", "sortDirection"])
 	},
+	{
+		dataKey: "shortTitle",
+		disabledIn: "feed",
+		submenu: true,
+		label: "zotero.items.shortTitle_column",
```

Comment:
@adomasven: Is there any good reason we can't use zotero.properties strings for these? Right now it [uses strings from `Zotero.Intl.strings`](https://github.com/zotero/zotero/blob/f333585d988f9c2d3b7241386d730466faf0ac36/chrome/content/zotero/components/virtualized-table.jsx#L1072-L1074), but I assume that's a vestige of the XUL tree needing strings to be in zotero.dtd. If we're getting them via JS anyway, we should just get them with `getString()` so that we can reuse zotero.properties strings. I'd rather not add another duplicate string to zotero.dtd that everyone will have to translate.

The one catch is that we're not explicitly differentiating between raw strings and string ids, and while `getString()` returns `false` if the string doesn't exist, it also logs an error. But we could just try `getString()` as long as the string looks like a key: e.g., `^[^\s]+\w\.\w[^\s]+$`

Reply:
I didn't realize we were duplicating these in both of the files. Yeah, we can switch to `getString()`.

Reply:
OK, done. I kept around `creator_column` and `year_column` because they don't have equivalents in `zotero.properties` and `dateModified_column` because it's localized as "Date Modified" instead of "Modified" as in the item pane. Is that right? And should those be moved to `zotero.properties`, or are they fine staying in the DTD? (I assume that's better so we don't have to retranslate them on Transifex.)